NineChronicles.Headless.Executable/Store/IStoreExtensions.cs

```diff
@@ -0,0 +1,28 @@
+#nullable enable
+using System;
+using System.Linq;
+using System.Security.Cryptography;
+using Libplanet;
+using Libplanet.Action;
+using Libplanet.Blocks;
+using Libplanet.Store;
+
+namespace NineChronicles.Headless.Executable.Store
+{
+    public static class IStoreExtensions
+    {
+        public static Block<T> GetGenesisBlock<T>(this IStore store)
+            where T : IAction, new()
+        {
+            var chainId = store.GetCanonicalChainId();
+            if (chainId is null)
+            {
+                throw new ArgumentException("The store doesn't have genesis block.", nameof(store));
```

Comment:
Originally, I thought it can be treated as *argument* because it seems the argument in syntax sight. After I see your review, I researched more and became to think [`InvalidOperationException`](https://docs.microsoft.com/en-us/dotnet/api/system.invalidoperationexception?view=netcore-3.1) will be more natural because the `store`'s state affects the method call. Do you agree?

> The exception that is thrown when a method call is invalid for the object's current state.

Reply:
Looks good to me too. 👍

Reply:
I applied it in 0bb7934.